AmplifyPlugins/API/AWSAPICategoryPlugin/AWSAPIPlugin.swift

```diff
@@ -38,6 +37,23 @@ final public class AWSAPIPlugin: NSObject, APICategoryPlugin {
     /// and is clearable by `reset()`. This is implicitly unwrapped to be destroyed when resetting.
     var subscriptionConnectionFactory: SubscriptionConnectionFactory!
 
+    /// The underlying map that retains instances of NetworkReachabilityNotifier.  Using a computed property
+    /// to work around @available for use on stored properties
+    var iReachabilityMap: [String: Any]?
```

Comment:
1. Prefer `private var _<nameOfVar>` to indicate a shadow var backing a property.
1. But I'm not sure you need it. If you're just defining a map of `[String: SomeProtocol]`, you should be fine, as long as you define `SomeProtocol` to have no availability restrictions at the top level, and only restrict the method:

```swift
protocol NetworkReachabilityNotifier {
    @available(iOS 13.0, *)
    func whatever()
}
```

Reply:
1.  I also prefer _<nameOfVar> , but when I tried to create _<nameOfVar> it gives me an error of:
```
Identifier Name Violation: Variable names should only contain alphanumeric characters: '_ReachabilityMap' (identifier_name)
```
2.  I am unable to follow you with regards to your suggestion with the NetworkReachabilityNotifier.  Lets sync up when we are both around.

Reply:
1. D'oh! Forgot about SwiftLint not liking underscore-prefixed symbols.